src/main/resources/reference.conf

```diff
@@ -17,4 +17,16 @@ videobridge {
     # be considered disconnected
     max-inactivity-limit=3 seconds
   }
+  # The APIs by which the JVB can be controlled
+  apis {
+    xmpp-client {
+      enabled=false
```

Comment:
i added this separate `enabled` value because i thought it would be nice to be able to disable these without having to clear all the env configurations.